packages/@aws-cdk/aws-gamelift-alpha/lib/fleet-base.ts

```diff
@@ -639,10 +639,10 @@ export abstract class FleetBase extends cdk.Resource implements IFleet {
   }
 
   protected warnVpcPeeringAuthorizations(scope: Construct): void {
-    cdk.Annotations.of(scope).addWarning([
+    cdk.Annotations.of(scope).addWarningV2('Gamelift:Fleet:AutorizeVpcPeering', [
```

Comment:
If we're going to schematize this, let's set a standard? This one uses a 3-part warning, the one below uses a 2-part warning.

If we take a cue from the feature flags, which this feels a bit similar to, maybe the flag format is something like `@aws-cdk/aws-service:someCamelCaseTerm` ?

Reply:
yeah I like this format better `@aws-cdk/aws-service:someCamelCaseTerm`. I'll update.